diff -durN CMFPlone-2.0-beta3.orig/CMFPlone/migrations/oneX_twoBeta2.py CMFPlone-2.0-beta3/CMFPlone/migrations/oneX_twoBeta2.py
--- CMFPlone-2.0-beta3.orig/CMFPlone/migrations/oneX_twoBeta2.py	Tue Oct 28 19:55:09 2003
+++ CMFPlone-2.0-beta3/CMFPlone/migrations/oneX_twoBeta2.py	Wed Nov 19 14:10:14 2003
@@ -34,6 +34,7 @@
         portal.manage_addProduct['CMFPlone'].manage_addTool('Portal Interface Tool')
 
     # add portal_prefs
+    skinsTool = getToolByName(portal, 'portal_skins')
     skins = skinsTool.getSkinSelections()
     for skin in skins:
         path = skinsTool.getSkinPath(skin)
